core/src/main/java/org/infinispan/configuration/global/GlobalConfigurationBuilder.java

```diff
@@ -157,6 +161,11 @@ public ShutdownConfigurationBuilder shutdown() {
       return modules;
    }
 
+   public GlobalConfigurationBuilder zeroCapacityNode(Boolean zeroCapacityNode) {
```

Comment:
Should be boolean, otherwise NPE will be thrown.

Reply:
yes